pkg/private/BUILD

```diff
@@ -24,6 +24,7 @@ filegroup(
     name = "standard_package",
     srcs = [
         "BUILD",
+        "install.py.in",
```

Comment:
Grumble.... We should have noticed this before. I have been naming all template files .tmpl and you have been naming them .in.  It all depends on your individual experience which seems right.

So I looked at the Bazel sources, and we seem to favor .tmpl there. There are a few .in files, but they are only under third_party, and all autoconf files. How do you feel about using .tmpl here? That follows the Bazel pattern and is distinctly not to be confused with a `configure` input.

Reply:
I'm fine with changing this to `.tmpl`, but I personally prefer the ".in" extension to allow for reduced redundancy (e.g. `template.spec.in` looks better to me than `template.spec.tmpl`).

All that being said, It looks like bazel itself is inconsistent, but seems to favor `.tpl` over `.tmpl`:

```
> find -name \*.tpl
./tools/osx/crosstool/BUILD.tpl
./tools/cpp/BUILD.tpl
./tools/cpp/osx_cc_wrapper.sh.tpl
./tools/cpp/vc_installation_error.bat.tpl
./tools/cpp/BUILD.empty.tpl
./tools/cpp/BUILD.toolchains.tpl
./tools/cpp/clang_installation_error.bat.tpl
./tools/cpp/BUILD.windows.tpl
./tools/cpp/linux_cc_wrapper.sh.tpl
./scripts/docs/jekyll_build.sh.tpl
./src/test/java/com/google/devtools/build/skydoc/testdata/generated_bzl_test/dep.bzl.tpl
```
```
> find -name \*.tmpl
./src/test/shell/bazel/testdata/jdk_http_archives.tmpl
./src/test/shell/testenv.sh.tmpl
./src/main/java/com/google/devtools/build/lib/bazel/rules/java/jdk.WORKSPACE.tmpl
```
```
> find -name \*.in
./third_party/py/mock/MANIFEST.in
./third_party/py/six/MANIFEST.in
./third_party/zlib/zconf.h.in
./third_party/zlib/contrib/minizip/minizip.pc.in
./third_party/zlib/zlib.pc.in
./third_party/zlib/Makefile.in
```

Tracked this in https://github.com/bazelbuild/rules_pkg/issues/382